pkgs/applications/window-managers/x-create-mouse-void/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, xorg, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "x-create-mouse-void";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "cas--";
+    repo = "XCreateMouseVoid";
+    rev = "0.1";
+    sha256 = "151pv4gmzz9g6nd1xw94hmawlb5z8rgs1jb3x1zpvn3znd7f355c";
+  };
+
+  buildInputs = [ xorg.libX11 ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -pv $out/bin
+    cp -av XCreateMouseVoid $out/bin/x-create-mouse-void
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/cas--/XCreateMouseVoid";
+    description =
```

Comment:
Here shouldn't be a line break.

Reply:
Applied (still had nixfmt configured in Emacs).